### ISSUE TITLE
Sc-compReg enables the comparison of gene regulatory networks between conditions using single-cell data

### ISSUE THREAD

**Issue by cmclean5** [cmclean5/PublicHealthModels#8]
## TL;DR

Paper propose as method 'sc-compReg' for the comparative analysis of gene expression regulatory networks between two conditions using single cell gene expression (scRNA-seq) and single cell chromatin accessibility data (scATAC-seq).

### Paper Link

https://www.nature.com/articles/s41467-021-25089-2

### Author/Institution

Zhana Duren (Clemson University, Stanford University)

## Overview

 Method applied to compare the gene regulatory networks of an individual with chronic lymphocytic leukemia (CLL) versus a healthy control. The analysis reveals a tumor-specific B cell subpopulation in the CLL patient and identifies TOX2 as a potential regulator of this subpopulation.

## Cite

Duren, Z., Lu, W.S., Arthur, J.G. et al. Sc-compReg enables the comparison of gene regulatory networks between conditions using single-cell data. Nat Commun 12, 4763 (2021). https://doi.org/10.1038/s41467-021-25089-2

## Comments
The sc-compReg software is freely available: https://github.com/SUwonglab/sc-compReg
